app/src/main/java/com/eveningoutpost/dexdrip/Models/BgReading.java

```diff
@@ -801,6 +805,10 @@ public static void checkForDropAllert(Context context) {
             Log.d("NOTIFICATIONS", "checkForDropAllert: Notifications are currently disabled!!");
             return;
         }
+        if(prefs.getLong("low_alerts_disabled_until", 0) > new Date().getTime()){
+            Log.d("NOTIFICATIONS", "checkForDropAllert: Low alerts are currently disabled!!");
```

Comment:
ok I will remove that check, drop alert indeed is something different than low alert